src/compositions/useThemeTokens.ts

```diff
@@ -1,14 +1,24 @@
 import { reactive, watch, readonly } from 'vue'
 import { useColorTheme } from '@/compositions/useColorTheme'
 
-function factory(): () => Readonly<ReturnType<typeof getTokens>> {
+type ThemeTokens = ReturnType<typeof getTokens>
+
+function factory(): () => Readonly<ThemeTokens> {
   const { value: theme } = useColorTheme()
 
-  const tokens = reactive(getTokens())
+  const tokens = reactive({}) as ThemeTokens
+
+  setTokens()
 
-  watch(theme, () => Object.assign(tokens, getTokens()))
+  watch(theme, () => setTokens())
+
+  function setTokens(): void {
+    requestAnimationFrame(() => {
+      Object.assign(tokens, getTokens())
+    })
```

Comment:
What's the thinking behind the animation frame here? I'd expect something more on the lines of:
```ts
document.addEventListener("DOMContentLoaded", () => {
   Object.assign(tokens, getTokens())
})
```

as a better proxy for loaded stylesheets but maybe I'm misunderstanding the problem